configuration.adoc

```diff
@@ -1701,7 +1701,14 @@ zmcontrol restart
 S/MIME is a standard to send secure email messages. S/MIME messages use
 digital signature to authenticate and encrypt messages.
 
-=== Prerequisites
+Currently, there are two versions of S/MIME feature
+
+. The old client based solution which requires Java 1.6 SE deployed on client machine
+. The new server based solution which does not require Java on client machine. The server performs all the cryptographic operations.
+
+=== Setting up client based S/MIME feature
```

Comment:
Suggest:
Setting up for using the S/MIME feature using the client base solution.